src/stemmabench/algorithms/utils.py

```diff
@@ -83,7 +82,44 @@ def find_root(dict_of_children: Dict[str, Any]) -> List[str]:
         return root
 
     @staticmethod
-    def dict_from_edge(*, edge_path: Union[str, None] = None, edge_list: Union[List[List[str]], None] = None) -> Dict[str, Any]:
+    def recursive_fit(input_dict: Dict[str, dict],
+                      to_be_fited_label: str,
```

Comment:
```suggestion
                      to_be_fitted_label: str,
```

everywhere also : fited -> fitted

Reply:
Done